libs/coin-modules/coin-cosmos/src/synchronisation.ts

```diff
@@ -102,7 +102,9 @@ const getBlankOperation = (tx: CosmosTx, fees: BigNumber, accountId: string): Co
     accountId,
     date: new Date(tx.timestamp),
     extra: {},
-    transactionSequenceNumber: parseInt(tx?.tx?.auth_info?.signer_infos[0]?.sequence || "0"),
+    transactionSequenceNumber: new BigNumber(
+      parseInt(tx?.tx?.auth_info?.signer_infos?.[0]?.sequence ?? "0"),
```

Comment:
You might want to keep the `||` because it means an empty string will not fallback to `"0"` now and I don't know what parseInt will return

We could also probably drop the parseInt as BigNumber already works with string